frappe/data_migration/doctype/data_migration_connector/connectors/frappe_connection.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import unicode_literals
+import frappe
+from frappe.frappeclient import FrappeClient, FrappeException
+from .base import BaseConnection
+
+class FrappeConnection(BaseConnection):
+	def __init__(self, connector):
+		self.connector = connector
+		self.connection = FrappeClient(self.connector.hostname,
+			self.connector.username, self.connector.password)
+
+	def insert(self, doctype, doc):
+		doc = frappe._dict(doc)
+		doc.doctype = doctype
+		try:
+			response_doc = self.connection.insert(doc)
```

Comment:
exception handling should be by caller since there is no value add by the connector